support-frontend/assets/helpers/user/user.js

```diff
@@ -149,6 +145,17 @@ const init = (dispatch: Function, actions: UserSetStateActions = defaultUserActi
     dispatch(setStateField(window.guardian.user.address4 || window.guardian.geoip.stateCode));
     dispatch(setIsSignedIn(true));
     dispatch(setEmailValidated(getEmailValidatedFromUserCookie(cookie.get('GU_U'))));
+
+    fetch(`${window.guardian.mdapiUrl}/user-attributes/me`, {
+      mode: 'cors',
+      credentials: 'include',
+    }).then(response => response.json())
+      .then((attributes) => {
+        if (attributes.recurringContributionPaymentPlan) {
+          dispatch(setIsRecurringContributor());
```

Comment:
the default value is false https://github.com/guardian/support-frontend/blob/5ddfca4d311ad85a3bfb3d32cba37353f0448246/support-frontend/assets/helpers/user/userReducer.js#L42
so that means if we have issues with mdapi or this call stops working, we will still allow recurring contributions
I think is is correct, just wanted to write it down to confirm.

Reply:
Yep, there are certainly cases where a user could create 2 recurring contribution, and this PR makes it more likely. But I think it's still pretty unlikely, and enabling users to retry is more important